cgi-bin/helpers/database.py

```diff
@@ -1,7 +1,8 @@
 from methods import ROOT, get_logger
 from urlparse import urlparse
 
-import json, os, psycopg2
+import json, os
+psychopg2 = None
```

Comment:
Could we do a `try` and `except ImportError: pass` here? Or, import it only `if os.environ.get('DATABASE_URL') is not None`. Looks a bit cleaner.